Update rule.md (#6479)
SSWConsulting/SSW.Rules.Content: rules/dotnet-upgrade-for-complex-projects/rule.md

```csharp
<TargetFrameworks>net472;net8.0</TargetFrameworks>
```

![Figure: Git changes for targeting to multiple target frameworks](target-to-multiple-TFMs.png)

# Upgrading

> At this point, ensure your project can target both the .NET Framework and the new target .NET. Some of the projects might not support both platforms right away and you can follow these steps to fix the issues and have a better understanding of how much work it might lies ahead.

Reviewer: ```suggestion
At this point, ensure your project can target both the .NET Framework and the new target .NET. Some of the projects might not support both platforms right away and you can follow these steps to fix the issues and better understand how much work might lie ahead.
```

A little Grammarly fix as "it might lies ahead" doesn't sound right, anymore.
